source/includes/api-reference/_api-reference.html.md

```diff
@@ -1,5 +1,12 @@
 # API Overview
 
+```
+##########
```

Comment:
Why do the hash marks end here as opposed to going to the end of the longest sentence?